shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultResult.java

```diff
@@ -16,31 +17,58 @@
  * Results for one fault computation.
  *
  * @author Boubakeur Brahimi
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
  */
 public final class FaultResult extends AbstractExtendable<FaultResult> {
 
-    private final String id;
+    private final Fault fault;
 
     private final double threePhaseFaultCurrent;
```

Comment:
After our talk, it seems that there is a kind of redondance between `threePhaseFaultCurrent` and `current`. I continue to think that it is useful to expose only the `threePhaseFaultCurrent`. @sylvlecl what do you think about the `FaultResult` design?